### PR TITLE
[ Sri ] : A 1206522048129442 | Refactor the patient list view in the provider acknowledgement flow

### PR DESCRIPTION
**Pull request by srinithishg**
Story no - 1206522048129442 

Patient with the earliest notification should be listed first.
Inside patient accordion, the earliest notification should be listed first.
When user clicks on the patient ID, targeted tab should open 
The way patient details are mentioned, should be similar to other pages

### CODE REVIEW

File: micro-frontends/src/next-ui/Components/ProviderNotificationPatients/PatientsList.spec.jsx [hunk 1]
```diff
@@ -2,30 +2,119 @@ import React from 'react';
 import { render, waitFor } from '@testing-library/react';
 import PatientsList from './PatientsList';
 
-const mockGetEmergencyDrugAcknowledgements = jest.fn().mockResolvedValue([]);
+const mockGetEmergencyDrugAcknowledgements = jest.fn();
+
+const mockData = [
```
Comment (abinaya-u) [inline micro-frontends/src/next-ui/Components/ProviderNotificationPatients/PatientsList.spec.jsx:7]: can we move mock datas to a separate file

File: micro-frontends/src/next-ui/Components/ProviderNotificationPatients/PatientsList.spec.jsx [hunk 2]
```diff
@@ -2,30 +2,119 @@ import React from 'react';
...
+    "administered_drug_name": "Paracetamol 250 mg Suppository",
+    "administered_dose": 1,
+    "administered_dose_units": "Tablet(s)",
+    "administered_route": "Oral",
+    "medication_administration_performer_uuid": "0e56149a-fc47-448f-bd14-f0c8cc5826c4",
+    "visit_uuid": "6f0c44e9-9555-45d2-acd5-8d6721029b67"
+  },
+]
+
+const sortedMockData = [
```
Comment (abinaya-u) [inline micro-frontends/src/next-ui/Components/ProviderNotificationPatients/PatientsList.spec.jsx:54]: this as well

File: micro-frontends/src/next-ui/Components/ProviderNotificationPatients/PatientsList.spec.jsx (hunk 1 above)
Comment (srinithishg) [reply micro-frontends/src/next-ui/Components/ProviderNotificationPatients/PatientsList.spec.jsx]: I've moved the data to seperate file

File: micro-frontends/src/next-ui/Components/ProviderNotificationPatients/PatientsList.spec.jsx (hunk 2 above)
Comment (srinithishg) [reply micro-frontends/src/next-ui/Components/ProviderNotificationPatients/PatientsList.spec.jsx]: I've moved the data to seperate file

**Review by abinaya-u — APPROVED** [review]

**Pull request merged**